common/vm_wait_guest_ip.yml

```diff
@@ -33,50 +28,40 @@
     datacenter: "{{ vsphere_host_datacenter }}"
     folder: "{{ vm_folder }}"
     name: "{{ vm_name }}"
-  delay: 5
+    schema: "vsphere"
+    properties: ['guest']
   retries: "{{ ((vm_get_ip_timeout | int) / 5) | int }}"
-  register: vm_guest_facts
+  register: vm_guestinfo
```

Comment:
Need to search if there are any other places using this variable `vm_guest_facts`

Reply:
Have searched. The `vm_guest_facts` defined here are used by `vm_get_ip_from_vmtools.yml`, which has been updated. For the other places using `vm_guest_facts`, they are defined in other common tasks.